stress_test.py

```diff
@@ -0,0 +1,444 @@
+                [self._cl, 'run', ':' + file.name(), 'python ' + file.name(), '--request-gpus=1']
+            )
+        file.delete()
+
+    def _create_infinite_memory_script(self):
+        code = 'a=["codalab stress test memory"]\nwhile True: a.extend(a); print(a)'
+        return TestFile('stress_memory.py', content=code)
+
+    def _test_infinite_disk(self):
+        if not self._should_test_infinite_disk():
+            return
+        self._set_worksheet('infinite_disk')
+        # Infinitely write out random characters to disk
+        self._run_bundle([self._cl, 'run', 'dd if=/dev/zero of=1g.bin bs=1G;'])
+        self._run_bundle([self._cl, 'run', 'dd if=/dev/urandom of=/dev/sda;'])
+
+    def _test_many_disk_writes(self):
+        self._set_worksheet('many_disk_writes')
+        for _ in range(self._get_disk_write_count()):
+            # Write out 1 GB worth of bytes out to disk
+            self._run_bundle([self._cl, 'run', 'dd if=/dev/zero of=1g.bin bs=1G count=1;'])
+
+    def _set_worksheet(self, run_name):
+        worksheet_name = self._create_worksheet_name(run_name)
+        uuid = run_command([self._cl, 'new', worksheet_name], force_subprocess=True)
+        run_command([self._cl, 'work', worksheet_name], force_subprocess=True)
+        run_command([self._cl, 'wedit', '--tag=%s' % StressTestRunner._TAG], force_subprocess=True)
+        return uuid
+
+    def _create_worksheet_name(self, base_name):
+        return 'stress_test_{}_ws_{}'.format(base_name, self._generate_random_id())
+
+    def _generate_random_id(self):
+        return ''.join(
+            random.choice(string.ascii_lowercase + string.ascii_uppercase) for _ in range(24)
+        )
+
+    def _run_bundle(self, args):
+        args.append('--tags=%s' % StressTestRunner._TAG)
+        return run_command(args, force_subprocess=True)
+
+    def cleanup(self):
+        if self._should_bypass_cleanup():
+            return
+        print('Cleaning up...')
+        bundles = run_command(
+            [self._cl, 'search', 'tags=%s' % StressTestRunner._TAG, '.limit=10000', '--uuid-only'],
+            force_subprocess=True,
+        ).split('\n')
+        worksheets = run_command(
+            [self._cl, 'wsearch', 'tag=%s' % StressTestRunner._TAG, '.limit=1000', '--uuid-only'],
+            force_subprocess=True,
+        ).split('\n')
+        for uuid in bundles:
+            if len(uuid) == 0:
+                continue
+            # Wait until the bundle finishes and then delete it
+            run_command([self._cl, 'wait', uuid], force_subprocess=True)
+            run_command([self._cl, 'rm', uuid, '--force'], force_subprocess=True)
+        for uuid in worksheets:
+            if len(uuid) == 0:
+                continue
+            run_command([self._cl, 'wrm', uuid, '--force'], force_subprocess=True)
+        print('Removed {} bundles and {} worksheets.'.format(len(bundles), len(worksheets)))
+
+    def _should_bypass_cleanup(self):
```

Comment:
Instead of these accessors, can't you just directly write `self._args.bypass_cleanup`?  Trying to minimize the amount of boilerplate...

Reply:
I got rid of the accessors, but I wrapped the args around an enum so I have to access the values a bit differently using a dict.

Reply:
If the preferred way is to use the namespace object directly, I can get rid of the enum as well.